sdk/core/core/src/az_credential.c

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include <az_credential.h>
+#include <az_result.h>
+
+#include <az_contract.h>
+
+#include <_az_cfg.h>
+
+AZ_NODISCARD az_result
```

Comment:
Wonder if we can INLINE this directly in H file, since it is short implementation